great_expectations/data_context/data_context/file_data_context.py

```diff
@@ -150,11 +150,8 @@ def _load_file_backed_project_config(
             raise gx_exceptions.ConfigNotFoundError()
 
         try:
-            return cast(
```

Comment:
Was able to get rid of this cast by using a `TypeVar` on the parent class of `DataContextConfig`

Reply:
I struggled with it at first because it was a class method.

I couldn't figure out how I could use a `TypeVar` for both a class and instance type... and then all of the sudden it became obvious.